lib/tasks/taxonomy/analysis/taxons_tagged_to_content.rake

```diff
@@ -24,21 +24,28 @@ namespace :taxonomy do
         page += 1
       end
 
-      puts %w(organisations content_id base_path number_of_taxons).to_csv
+      content_items.each do |item|
+        details = []
+        item["taxons"].each do |taxon|
+          resp = Services.publishing_api.get_content(taxon)
+          details << resp["base_path"]
+        end
+        item["base_paths"] = details
+      end
 
-      content_items.each do |content_item|
-        organisations = content_item.fetch('organisations', [])
-          .map { |org| org['title'] }
-          .join('; ')
+      puts %w(Link Number_of_taxons Base_path_1 Taxon_id_1 Base_path_2 Taxon_id_2 Base_path_3 Taxon_id_3).to_csv
 
+      content_items.each do |content_item|
         csv_row = [
-          organisations,
-          content_item['content_id'],
           content_item['link'],
           content_item['taxons'].length,
         ]
 
-        content_item['taxons'].each { |taxon_id| csv_row << taxon_id }
+        taxon_details = content_item['base_paths'].zip(content_item['taxons']).flatten
+
+        taxon_details.each do |taxon_detail|
```

Comment:
@alecgibson: Wouldn't this add taxon_details as an array? I want the details to appear in separate columns